ext/zip/php_zip.stub.php

```diff
@@ -641,6 +641,9 @@ class ZipArchive implements Countable
     /** @tentative-return-type */
     public function open(string $filename, int $flags = 0): bool|int {}
 
+    /** @tentative-return-type */
```

Comment:
New methods doesn't need to have tentative return type. You can freely use a native type declaration.

Reply:
I have removed it :)